docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -118,13 +211,13 @@ On CentOS:
 
 RPMs for RHEL 7 and RHEL 8 are available from the `Ansible Engine repository <https://access.redhat.com/articles/3174981>`_.
 
-To enable the Ansible Engine repository for RHEL 8, run the following command:
+To enable the Ansible repository for RHEL 8, run the following command:
```

Comment:
shouldn't the Engine stay here (and below)?

Reply:
Ah, you're right. I was thinking about the GitHub repository, but this is referring to something different.

Reply:
Addressed in 28719122db32af5ccbbf9944cb1cf04ffc8f4d10